### ISSUE TITLE
Use <footer> landmark

### ISSUE THREAD

**Issue by nrsa222** [uklibraries/UKL_HeaderFooter#90]
**Issue**
Since the universal footer should be the very last thing on every page, semantically it should be included in a `<footer>` tag.  Currently the html is wrapped in `<div role="contentinfo" aria-label="University footer" class="ukl-ext-clamp">`.

There are many sites we're injecting this code into without checking to see if there is already a `<footer>` tag defined.  Having content or regions after a `<footer>` tag is frowned upon and having two `<footer>` tags doesn't break the html but is messy.  This means that each site injecting the UKL footer will need to be manually checked and the html markup changed if there is already a `<footer>`, to implement this version.

These proposed changes to the footer can also lead to a scenario where users can implement the required global footer content as they see fit per their site and pull the the CSS from the CDN, but then they become responsible for maintaining the content in the footer.

Relates to #88 and #89 
 
**Solution**

- [x] Update readme to explain manual checks that need to be made before implementing this version.
- [x] Update comboFooterShared.js to replace `<div role="contentinfo">` with `<footer role="contentinfo">`
